scipy/odr/__init__.py

```diff
@@ -128,6 +124,7 @@ def f(B, x):
 
 from .odrpack import *
 from .models import *
+from . import add_newdocs as _add_newdocs  # for odr.odr() docstring import
```

Comment:
This was actually already visible in the namespace (e.g. with `dir(odr)` or `odr.<tab>`). It's a small thing, but now that we're fixing this here why not clean this up as well. How about changing the filename to `_add_newdocs.py` and then here `from . import _add_newdocs`?

Reply:
Good idea. I did it.